src/isyntax/isyntax.c

```diff
@@ -1774,14 +1774,13 @@ u32 isyntax_idwt_tile_for_color_channel(isyntax_t* isyntax, isyntax_image_t* wsi
 	return invalid_edges;
 }
 
-u32* isyntax_load_tile(isyntax_t* isyntax, isyntax_image_t* wsi, i32 scale, i32 tile_x, i32 tile_y) {
+u32* isyntax_load_tile(isyntax_t* isyntax, isyntax_image_t* wsi, i32 scale, i32 tile_x, i32 tile_y, block_allocator_t* ll_coeff_block_allocator, bool decode_rgb) {
```

Comment:
I need explicit block allocator passing because block allocators can now be shared between isyntax_t instances. This is to allow eviction of "another isyntax's tiles".

decode_rgb flag is needed to avoid decoding rgb for intermediate tiles. I know slidescape uses and caches those, so you may be calling this with decode_rgb=true always.

Reply:
OK!
One question regarding the sharing of block allocators for multiple isytaxes: will this still work if the codeblock size is different? (Maybe not so likely to happen, but I think the spec allows for this)

Reply:
> will this still work if the codeblock size is different

Not as-is, I have a todo in OpenSlide code, but I currently assume all iSyntax use same block size. I will add some asserts to make sure the shared block allocator is of the right size at runtime.